devcontainer/devcontainer.go

```diff
@@ -258,21 +258,34 @@ func UserFromDockerfile(dockerfileContent string) string {
 // ImageFromDockerfile inspects the contents of a provided Dockerfile
 // and returns the image that will be used to run the container.
 func ImageFromDockerfile(dockerfileContent string) (name.Reference, error) {
+	args := map[string]string{}
+	var imageRef string
 	lines := strings.Split(dockerfileContent, "\n")
 	// Iterate over lines in reverse
 	for i := len(lines) - 1; i >= 0; i-- {
 		line := lines[i]
-		if !strings.HasPrefix(line, "FROM ") {
+		if strings.HasPrefix(line, "ARG ") {
```

Comment:
It seems like technically leading whitespace is allowed. See "Note on whitespace" on https://docs.docker.com/engine/reference/builder/.

I'm not sure if multiple spaces (or tabs?) after `ARG` or `FROM` are permitted... but in case they are, tokenizing on whitespace might be a better way to do this.